securitas/themes/fas/static/fonts/font-awesome.css

```diff
@@ -14,6 +14,7 @@
 .fa {
   display: inline-block;
   font: normal normal normal 14px/1 FontAwesome;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
What is this line doing? This is for an icon font, it's none of those families, right?

Reply:
Additionally, this basically an bundled library. so im -1 for changing this file